nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -523,13 +550,18 @@ def __init__(
         self._pruned_module_groups_info = pruned_module_groups_info
         self._transformation_commands = transformation_commands
         self._filter_importance_fn = filter_importance_fn
+        self._filter_importance = None
+        if filter_importance_path is not None:
+            self._filter_importance = torch.load(filter_importance_path)
```

Comment:
Reading from file introduces dependency from the file system and defines not trivial interface: file should be loaded by torch and be in a specific format. 

Please make it more explicit. Here's the example where bitwidth for mixed precision is defined per nncf scope. 
https://github.com/openvinotoolkit/nncf/blob/c732de08a14ce389a3a435fc70d53ee9ebd91564/nncf/torch/quantization/precision_init/manual_init.py#L27
https://github.com/openvinotoolkit/nncf/blob/c732de08a14ce389a3a435fc70d53ee9ebd91564/examples/torch/classification/configs/mixed_precision/mobilenet_v2_imagenet_mixed_int_manual_staged.json#L38

Reply:
Here's a json schema for the reference: https://github.com/openvinotoolkit/nncf/blob/c732de08a14ce389a3a435fc70d53ee9ebd91564/nncf/config/schemata/algo/quantization.py#L294

Reply:
`external_importance_path` is just a string path, we need to load it to get the importance information, rather than writing the information directly into the config file, so it seems that the bitwidth_per_scope example provided for reference doesn't work here. 

Maybe we can refer to:
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/config/schemata/algo/quantization.py#L310-L317

Thanks!